src/sql/parts/dashboard/common/dashboardHelper.ts

```diff
@@ -166,16 +166,35 @@ export function addContext(config: WidgetConfig[], collection: any, context: str
  * Returns a filtered version of the widgets passed based on edition and provider
  * @param config widgets to filter
  */
-export function filterConfigs<T extends { when?: string }, K extends { contextKeyService: IContextKeyService }>(config: T[], collection: K): Array<T> {
+export function filterConfigs<T extends { provider?: string | string[], when?: string }, K extends { contextKeyService: IContextKeyService }>(config: T[], collection: K): Array<T> {
 	return config.filter((item) => {
-		if (!item.when) {
+		let when = item.when;
```

Comment:
nit: not sure why we moved this to a local variable since it's only accessed once on line 175

Reply:
Thanks - meant to remove this. Was leftover from my 1st attempt where I wanted to change the "when" condition. This turned out to be non-viable due to the limitation of the `when` clause for complex conditions.